### DISCUSSION TITLE
Support for Overwritting all options on the commandline

### DISCUSSION THREAD

**Discussion by rdorsch** [Motion-Project/motion#1500] [General]
0. Reviewed guide and contributing documents? (Yes/No): Yes
1. version [x.y.z, hash, other]: 4.2.2
2. installed as a package or compiled from sources [deb, rpm, git, other]: deb of Debian sid
3. standalone or part of third party [motion, MotionEyeOS, other]: motion (standalone)
4. video stream source [V4L (card or USB), net cam (mjpeg, rtsp, other), mmal]: net cam
5. hardware [x86, ARM, other]: x86
6. operating system [16.04, Stretch, etc, FreeBSD, other]: Debian 10

I found that netcam_url supports file:// urls

    netcam_url file:///home/user/cam1/cam1_20180817084027.mkv

which is as it is correctly stated, super useful for "reprocessing" a movie created by Motion to fine tune the detection options. 

Many thanks for this. 

To make the process even better, it would be beneficial to be able to overwrite all config options on the motion command line, e.g.

motion -o netcam_url=file:///home/user/cam1/cam1_20180817084027.mkv

Thanks
Rainer

**Comment by pataquets** [comment]
+1. I'm not sure if it's true 100% of the time, but from my usage experience, I'd expect command-line provided options to override config-file ones.
